service/scaladsl/client/src/main/scala/com/lightbend/lagom/scaladsl/client/ServiceClient.scala

```diff
@@ -148,7 +148,9 @@ abstract class LagomClientApplication(
 
   override lazy val serviceInfo: ServiceInfo = ServiceInfo(clientName, Map.empty)
   override lazy val environment: Environment = Environment(new File("."), classLoader, Mode.Prod)
-  lazy val configuration: Configuration = Configuration.load(environment, Map.empty)
+  lazy val configuration: Configuration = Configuration.load(environment, Map(
+    "allowMissingApplicationConf" -> true
+  ))
```

Comment:
Sorry that I missed the compile error in Travis, but this needs to call a different overloaded version of `Configuration.load`:

```scala
  private[play] def load(
    classLoader: ClassLoader,
    properties: Properties,
    directSettings: Map[String, AnyRef],
    allowMissingApplicationConf: Boolean): Configuration
```

I think this could be replaced with something like:

```scala
Configuration.load(
  environment.classLoader,
  System.getProperties,
  Map.empty,
  allowMissingApplicationConf = true
)
```

It's probably a good idea to write a test, as well (one that fails without the fix).